### PR TITLE
Fix for bugs #2536 and #5024

### PR DESCRIPTION
**Pull request by keesd**
Fixed the Orchard behavior of passing windows credentials to e-mail server when no credentials where entered.

 Added an option to the smptsettings configuration to explicitly configure windows authentication.
This fixes bugs #2536 and #5024.

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Email/Models/SmtpSettingsPart.cs [hunk 1]
```diff
@@ -41,6 +41,12 @@ public class SmtpSettingsPart : ContentPart {
             set { this.Store(x => x.RequireCredentials, value); }
         }
 
+        public bool UseDefaultCredentials
+        {
```
Comment (sfmskywalker) [inline src/Orchard.Web/Modules/Orchard.Email/Models/SmtpSettingsPart.cs]: Please use K&R style formatting (opening braces on the same line).

**Comment by sfmskywalker** [comment]
The fix looks great and I like the added feature to properly use the ```UseDefaultCredentials``` (Use Windows authentication). However, I'd like to see the two radio buttons right underneath each other, and toggling the credentials input fields when the user selects the "Specify username/password" option. This can be done without writing javascript when using the ```data-controllerid``` attribute on a container element surrounding the username and password inputs (the value of that attribute would be the ID of the linked radio button). The script enabling this data- attribute comes from the "ShapesBase" script resource.

**Comment by keesd** [comment]
I know, but the radiobuttons are rendered with a name-attribute that contains a ‘ .’ (dot). This causes problems in the .js code that does the showing/hiding based on the data-controllerids.

This would require a fix in base.js, line 178
$("[name=" + controller.attr("name") + "]").click(function () { $("[name=" + $(this).attr("name") + "]")

the $[name] query would require quotes around the controller.attr("name") . 

Can I include this fix and reissue the pull request?

**Comment by sebastienros** [comment]
Please target 1.x, or dev if 1.x is no more.

**Comment by sebastienros** [comment]
Ping ?

**Comment by keesd** [comment]
resubmitted in pr #5257

**Comment by keesd** [comment]
Resubmitted in dev branch

**Pull request closed without merging**